src/main/webapp/feedback.js

```diff
@@ -37,3 +37,8 @@ function getScheduledInterviewId() {
 function getRole() {
   return new URLSearchParams(window.location.search).get('role'); 
 }
+
+// Adds the scheduledInterviewId to the request when the form is submitted.
+function doSubmit() {
```

Comment:
Update the function name to convey what it's doing, rather than trying to do that in a code comment.

Reply:
Done